### ISSUE TITLE
Feature Request: Have line numbers display in the RowHeader of the various grids

### ISSUE THREAD

**Issue by ksumrall** [paolosalvatori/ServiceBusExplorer#716]
When looking through large sets of messages, it's often helpful to have line numbers for the row as in the following image:

![image](https://user-images.githubusercontent.com/2331576/230464277-b4f27f4b-b9a5-40d2-9aca-ed45eda26396.png)

I'm not that familiar with windows UI applications so I didn't make a PR for the changes. I'll explain what I've done to achieve the above results instead. If the explanation seems reasonable, I'd be happy to create a PR.

In the following 2 files:
> C:\_repositories\github\paolosalvatori\ServiceBusExplorer\src\ServiceBusExplorer\Controls\HandleQueueControl.cs
> C:\_repositories\github\paolosalvatori\ServiceBusExplorer\src\ServiceBusExplorer\Controls\HandleSubscriptionControl.cs

I have place the following code (Based on a SO answer [here](https://stackoverflow.com/a/30793442/1930432)):
```c#
        private void dataGridView_RowPostPaint(object sender, DataGridViewRowPostPaintEventArgs e)
        {
            var grid = sender as DataGridView;
            var rowIdx = (e.RowIndex + 1).ToString();

            var centerFormat = new StringFormat()
            {
                Alignment = StringAlignment.Far,

                LineAlignment = StringAlignment.Center
            };

            Size textSize = TextRenderer.MeasureText(rowIdx, this.Font);

            if (grid.RowHeadersWidth < textSize.Width + 20)
            {
                grid.RowHeadersWidth = textSize.Width + 20;
            }

            var headerBounds = new Rectangle(e.RowBounds.Left, e.RowBounds.Top, grid.RowHeadersWidth - 4, e.RowBounds.Height);
            e.Graphics.DrawString(rowIdx, this.Font, SystemBrushes.ControlText, headerBounds, centerFormat);
        }
```

I then went into the UI designer and found the grids I know about and bound their RowPostPaint event to the new function.

As far as I can tell, that did the trick to achieved the results of having line numbers added to the rows.

**Comment by ksumrall** [comment]
@SeanFeldman I have done as you suggested and created a PR.
I'm not familiar with all that this tool offers, but, it looks like there may be other grids that may benefit from a row number display feature. It may be worth considering creating a subclass control to handle this and other common features.

**Comment by ErikMogensen** [comment]
Fixed in #717 😄
